### PR TITLE
new script to plot topomaps from yaml files

### PR DESCRIPTION
**Pull request by gpdas**
1. plot_yaml.py - new script to plot topomaps from yaml files
Usage:
```
rosrun topological_utils plot_yaml.py ~/rasberry_ws/src/RASberry/rasberry_navigation/maps/riseholme/Riseholme_06-2019.tmap ~/riseholme.png 12 12
```

2. minor modification to plot_topo_map.py

### CODE REVIEW

File: topological_utils/scripts/plot_yaml.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        ax.set_ylabel("Y")
+        ax.set_xlim(min_x - x_delta, max_x + x_delta)
+        ax.set_ylim(min_y - y_delta, max_y + y_delta)
+        fig.savefig(self.fig_file)
+
+if __name__ == "__main__":
+    rospy.init_node("plot_yaml")
+
+    if len(sys.argv) < 5:
+        rospy.loginfo("Usage is plot_yaml <path_to_yaml_topomap> <path_to_plot_file> <fig_width_inches> <fig_height_inches>")
```
Comment (Jailander) [inline topological_utils/scripts/plot_yaml.py:112]: Inches?!?!
bitmoji

Comment (gpdas) [reply topological_utils/scripts/plot_yaml.py:112]: Yes of course, what else do you expect in the GB? 
matplotlib figure dimensions are in inches. ;)

File: topological_utils/scripts/plot_yaml.py [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+                         visible=True)
+
+        ax.set_xlabel("X")
+        ax.set_ylabel("Y")
+        ax.set_xlim(min_x - x_delta, max_x + x_delta)
+        ax.set_ylim(min_y - y_delta, max_y + y_delta)
+        fig.savefig(self.fig_file)
+
+if __name__ == "__main__":
+    rospy.init_node("plot_yaml")
```
Comment (Jailander) [inline topological_utils/scripts/plot_yaml.py]: Isn't it better to use it without roscore? I think these kind of utilities can be used ros-free

**Comment by Jailander** [comment]
Looks really good, I found a small bug on which nodes with names that are not WayPointXXX get removed part of their name

**Comment by gpdas** [comment]
can you give an example?

**Comment by gpdas** [comment]
"WayPoint" is removed only if the full string is in the node name. 
e.g. WayPointXXX -> XXX
 Way_Point_XXX -> Way_Point_XXX (no change)

**Comment by Jailander** [comment]
Works great!!! will merge

**Pull request merged**